src/single-movie/SingleMovie.js

```diff
@@ -33,6 +35,24 @@ export default class SingleMovie extends Component {
     this.fetchSingleMovieData(this.props.movieID)
   }
 
+  rateMovie(event) {
+    if(Object.keys(this.props.userStatus).length > 0) {
+      this.setState({ toggle: true })
+    } else {
+      alert("You have to login in order to give your professional opinion about this movie")
+    }
+  }
+
+  displaySingleMovieRating(movieId) {
+    let ratings = this.props.movieRatings
+     let userMovieRating = ratings.find(rating => {
+        return parseInt(movieId) === rating.movie_id
```

Comment:
Smart call with the parseInt()! That was driving me nuts trying to figure out why nothing was returned